### PR TITLE
Lint everything

### PR DESCRIPTION
**Pull request by MatApple**
Now lint all `.js`. Lint errors will also now cause `npm test` to fail.

### CODE REVIEW

File: src/js/plugin-bridge/__tests__/PluginSDK-test.js [hunk 1]
```diff
@@ -34,71 +34,71 @@ describe('PluginSDK', function () {
...
-            // Don't return anything
-          }
-        );
-        PluginTestUtils.loadPlugins({
-          fakePlugin1: {
-            module: this.mockPlugin,
-            config: {
-              enabled: true,
-              foo: 'bar'
+          // Mock a fake plugin
```
Comment (mlunoe) [inline src/js/plugin-bridge/__tests__/PluginSDK-test.js]: This is just an indentation change, right?

File: tests/regressions/OverlayHeaderHeight-cy.js [hunk 2]
```diff
@@ -16,30 +16,30 @@ describe('Overlay Header Height [00j]', function () {
...
-      var overlayHeaderHeight = null;
-
-      // Open the Marathon task.
-      cy.get('@tableRows').contains('marathon').click();
-      // Get the task panel's header.
-      cy.get('.side-panel-header-container').as('sidePanelHeader');
-
-      cy.get('@sidePanelHeader').should(function ($sidePanelHeader) {
-        sidePanelHeaderHeight = $sidePanelHeader.outerHeight();
+        var sidePanelHeaderHeight = null;
```
Comment (mlunoe) [inline tests/regressions/OverlayHeaderHeight-cy.js]: Again, this is just an indentation change, right?

**Comment by mlunoe** [comment]
Assuming the answer to those two comments are 'yes', this is beautiful! Thank you! This is going to make my editor so happy 🎉 🎈 
:shipit:

**Comment by MatApple** [comment]
@mlunoe yes only indentation changes and updated lint task so `npm test` will fail on lint errors.

**Pull request merged**